tensorflow/compiler/xla/xla.proto

```diff
@@ -424,7 +424,11 @@ message DebugOptions {
   // (much) faster on our hardware.  Set this flag to disable this behavior.
   bool xla_cpu_strict_dot_conv_math = 175;
 
-  // Next id: 179
+  // An option to enable using cuDNN runtime compiled fusion kernels which is
+  // available and recommended for Ampere+ GPUs.
+  bool xla_gpu_use_runtime_fusion = 179;
```

Comment:
Probably better to explicitly set to false ? Do you intend to make it true by default?